lib/_emerge/EbuildPhase.py

```diff
@@ -136,6 +136,7 @@ async def _async_start(self):
 
             msg = []
             msg.append("Package:    %s" % self.settings.mycpv)
+            msg.append("Slot:       %s" % self.settings["SLOT"])
```

Comment:
Perhaps we could print it as part of 'package' instead?

Reply:
I'd personally prefer atom-style `%s:%s`.

Reply:
done